src/main/java/com/ecore/roles/web/rest/RolesRestController.java

```diff
@@ -53,14 +61,30 @@ public ResponseEntity<List<RoleDto>> getRoles() {
     }
 
     @Override
-    @PostMapping(
+    @GetMapping(
             path = "/{roleId}",
             produces = {"application/json"})
     public ResponseEntity<RoleDto> getRole(
             @PathVariable UUID roleId) {
         return ResponseEntity
                 .status(200)
-                .body(fromModel(rolesService.GetRole(roleId)));
+                .body(fromModel(rolesService.getRole(roleId)));
     }
 
+    @Override
+    @GetMapping(
+            path = "/search",
+            produces = {"application/json"})
+    public ResponseEntity<List<RoleDto>> getRolesByUserAndTeam(
```

Comment:
This method has been created to return the list of roles by userId and teamId raised by business rule.